lib/sport_ngin_aws_auditor/commands/audit.rb

```diff
@@ -7,6 +7,7 @@
   c.switch [:r, :reserved], :desc => "Shows reserved instance counts"
   c.switch [:i, :instances], :desc => "Shows current instance counts"
   c.flag [:t, :tag], :default_value => "no-reserved-instance", :desc => "Read a tag and group separately during audit"
+  c.flag [:h, :config_hash], :default_value => nil, :desc => "Print the audit according to this config hash instead of to config file"
```

Comment:
Hmm this should somehow describe that it's json. I sort of think we should change the name maybe `config_json` or `j`